lib/adapters/test-result/testplane.ts

```diff
@@ -32,7 +32,7 @@ export const getStatus = (eventName: ValueOf<Testplane['events']>, events: Testp
     } else if (eventName === events.TEST_PENDING) {
         return TestStatus.SKIPPED;
     } else if (eventName === events.RETRY || eventName === events.TEST_FAIL) {
-        return hasFailedImages(testResult.assertViewResults as ImageInfoFull[]) ? TestStatus.FAIL : TestStatus.ERROR;
+        return isAssertViewError(testResult.err) ? TestStatus.FAIL : TestStatus.ERROR;
```

Comment:
Test result with errors could still has failed images

Reply:
Why you check here only assert view error and don't check diff (`isImageDiffError`) or no ref image (`isNoRefImageError`)?

Reply:
Because testResult can only have assert view error.
`isImageDiffError` and `isNoRefImageError` - are image errors, and not the result ones. So if testResult has an image with `isImageDiffError` or `isNoRefImageError` - it would have `AssertViewError`